ipn/ipnauth/ipnauth.go

```diff
@@ -191,6 +192,36 @@ func (ci *ConnIdentity) IsReadonlyConn(operatorUID string, logf logger.Logf) boo
 	return ro
 }
 
+// IsLocalAdmin reports whether the connected user is considered a "local
+// admin". This is a higher privilege level than simple write access (per
+// IsReadonlyConn).
+//
+// Local admin means:
+//
+//   - Windows: member of the Administrators group (not necessarily The
+//     Administrator user, but can elevate privileges as needed)
+//   - macOS: member of the "admin" group
```

Comment:
I think? you can, and if so should, treat macOS and Linux the same. The admin group is special on macOS only in that it has the relevant entry in /etc/sudoers:

```
%admin          ALL = (ALL) ALL
```

In principle, macOS admins might change it to add groups and users, or whatever.